chatbot-core/backend/Dockerfile

```diff
@@ -1,21 +1,21 @@
 # Use the official Python base image
 FROM python:3.11.10-alpine
 
-# Set the working directory inside the container
-WORKDIR /app
+# Set label to push to GitHub Container Registry
+LABEL org.opencontainers.image.source "https://github.com/TinhHoaSolutions-EzHR/chatbot"
 
-# Copy the requirements file to the working directory
-COPY requirements.txt .
+# Install uv
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /uvx /bin/
 
-# Install the Python dependencies
-RUN pip install --upgrade pip && \
-    pip install --no-cache-dir -r requirements.txt
+# Copy the application into the container, including uv.lock
+COPY . /app
 
-# Copy the rest of the application code to the working directory
-COPY . .
+# Install dependencies
+WORKDIR /app
+RUN uv sync --frozen --no-cache
 
 # Expose the port on which the application will run
 EXPOSE 5000
 
 # Run the application
-CMD [ "uvicorn", "main:app", "--host", "0.0.0.0", "--port", "5000" ]
\ No newline at end of file
+CMD [ "/app/.venv/bin/fastapi", "run", "app/main.py", "--host", "0.0.0.0", "--port", "5000" ]
```

Comment:
Change to command `uv run uvicorn main:app --reload --port 5000`.

Reply:
https://github.com/TinhHoaSolutions-EzHR/chatbot/pull/49/commits/8467b47af2ba86a562c1d2b9ba2c0af763469260